### ISSUE TITLE
flaky required to run tests

### ISSUE THREAD

**Issue by stevepiercy** [Pylons/deformdemo#60]
It appears that `flaky` must be installed to run tests.

```
Steves-iMac:deformdemo stevepiercy$ env/bin/nosetests
<snip>
  File "/Users/stevepiercy/projects/deformdemo/deformdemo/test.py", line 15, in <module>
    from flaky import flaky
ModuleNotFoundError: No module named 'flaky'

----------------------------------------------------------------------
Ran 1 test in 0.265s

FAILED (errors=1)
Steves-iMac:deformdemo stevepiercy$ env/bin/pip install flaky
Collecting flaky
  Using cached https://files.pythonhosted.org/packages/fe/12/0f169abf1aa07c7edef4855cca53703d2e6b7ecbded7829588ac7e7e3424/flaky-3.6.1-py2.py3-none-any.whl
Installing collected packages: flaky
Successfully installed flaky-3.6.1
Steves-iMac:deformdemo stevepiercy$ env/bin/nosetests
Selenium says no
1569715494417   addons.manager  DEBUG   Application has been upgraded
# much happiness ensues
<snip>
```

Should I add it to [`testing` in `setup.py`](https://github.com/Pylons/deformdemo/blob/master/setup.py#L82)?

**Comment by stevepiercy** [comment]
Yes, Steve, you should. Incoming!

**Comment by stevepiercy** [comment]
Closed by #68
